tests/run_tests/driver_mt_test.py

```diff
@@ -25,8 +25,8 @@
 
 target_run = {
     "data_norm": 0.01577,
-    "phi_d": 12.04,
-    "phi_m": 0.2547,
+    "phi_d": 11.52,
```

Comment:
I guess these are changing because something was updated in the new SimPEG version we're pointing to in the pyproject.toml?  What changed?